applications/nrf5340_audio/src/audio/streamctrl.h

```diff
@@ -26,10 +27,8 @@ uint8_t stream_state_get(void);
 /**
  * @brief Send audio data over the stream.
  *
- * @param data		Data to send.
- * @param size		Size of data.
- * @param num_ch	Number of audio channels.
+ * @param audio_frame		Pointer to the audio data to send.
  */
-void streamctrl_send(void const *const data, size_t size, uint8_t num_ch);
+void streamctrl_send(struct audio_data *audio_frame);
```

Comment:
Should this be 
```suggestion
void streamctrl_send(const struct audio_data *audio_frame);
```

Or can/should the function be allowed to modify the frame before sending?